src/styles/miscStyles.js

```diff
@@ -83,6 +83,7 @@ export default ({ color, backgroundColor, borderColor, cardColor, dividerColor }
   label: {
     color,
     fontSize: 15,
+    textAlign: 'left',
```

Comment:
Hmmm, is this line also needed?
It is used in React-Native land and not in the message list.

Reply:
If I don't do that, the streams will appear like this:

![screenshot_1534757014](https://user-images.githubusercontent.com/12458563/44332159-a244ff00-a480-11e8-8a8b-dd3820147609.png)

It looks a bit weird that RTL stream names are shown to the other side, so I changed it so that they are shown beside the # mark.